lib/db/mysql/index.js

```diff
@@ -148,13 +152,20 @@ const QUERY_CODE_INSERT =
   'INSERT INTO codes (clientId, userId, email, scope, authAt, code) ' +
   'VALUES (?, ?, ?, ?, ?, ?)';
 const QUERY_TOKEN_INSERT =
-  'INSERT INTO tokens (clientId, userId, email, scope, type, token) VALUES ' +
-  '(?, ?, ?, ?, ?, ?)';
+  'INSERT INTO tokens (clientId, userId, email, scope, type, expiresAt, ' +
+  'token) VALUES (?, ?, ?, ?, ?, ?, ?)';
+const QUERY_REFRESH_TOKEN_INSERT =
+  'INSERT INTO refreshTokens (clientId, userId, email, scope, token) VALUES ' +
+  '(?, ?, ?, ?, ?)';
 const QUERY_TOKEN_FIND = 'SELECT * FROM tokens WHERE token=?';
```

Comment:
It's extra churn, but it may be worth renaming these to `QUERY_ACCESS_TOKEN_FIND` and similar in the interests of overall clarity.